pycbc/neutron_stars/pg_isso_solver.py

```diff
@@ -326,27 +298,19 @@ def PG_ISSO_solver(chi, incl):
         the radius of the orbit in BH mass units
     """
     # Auxiliary variables
-    cos_incl = np.cos(incl)
-    sgnchi = np.sign(cos_incl)*chi
-    if np.isscalar(sgnchi):
-        sgnchi = np.array(sgnchi, copy=False, ndmin=1)
+    if np.isscalar(chi):
         chi = np.array(chi, copy=False, ndmin=1)
         incl = np.array(incl, copy=False, ndmin=1)
-
+    chi = np.abs(chi)
     # ISCO radius for the given spin magnitude
-    initial_guess = [2 if s > 0.99 else (9 if s < 0 else 5) for s in sgnchi]
-    rISCO_limit = np.array([
-        root_scalar(
-            ISCO_eq, x0=g0, fprime=ISCO_eq_dr, fprime2=ISCO_eq_dr2,
-            args=(sc)).root
-        for g0, sc in zip(initial_guess, sgnchi)])
+    rISCO_limit = ISCO_solution(chi, incl)
     # If the inclination is 0 or pi, just output the ISCO radius
     equatorial = np.isclose(incl, 0) | np.isclose(incl, np.pi)
     if all(equatorial):
         return rISCO_limit
 
     # ISSO radius for an inclination of pi/2
-    initial_guess = [9 if c < 0 else 6 for c in chi]
+    initial_guess = [5.27451056440629 if c > 0.5 else 6 for c in chi]
```

Comment:
Can you insert a comment on the origin of this number?

Reply:
And at line 320: `polar = np.isclose(incl, np.pi / 2)` --> `polar = np.isclose(incl, 0.5*np.pi)`